docs/content/posts/2024-03-xx-release-1.19.0.md

```diff
@@ -0,0 +1,456 @@
+---
```

Comment:
@lincoln-lil I think we're missing something on the changes to the Sink API, from both https://issues.apache.org/jira/browse/FLINK-25857 and https://issues.apache.org/jira/browse/FLINK-33972. Same for https://cwiki.apache.org/confluence/display/FLINK/FLIP-400%3A+AsyncScalarFunction+for+asynchronous+scalar+function+support.

LGTM otherwise

Reply:
@MartijnVisser As we discussed at the sync meeting, I've added these descriptions (with the minor tweak of using 'Connector API Improvements' as the title, rather than the broader 'SDK Improvements').